src/jdk.management/share/classes/com/sun/management/doc-files/threadDump.schema.json

```diff
@@ -0,0 +1,99 @@
+        },
+        "runtimeVersion": {
+          "type": "string",
+          "description": "The runtime version, see java.lang.Runtime.version()."
+        },
+        "threadContainers": {
+          "type": "array",
+          "description": "The array of thread containers.",
+          "items": [
+            {
+              "type": "object",
+              "properties": {
+                "container": {
+                  "type": "string",
+                  "description": "The container name. The container name is unique."
+                },
+                "parent": {
+                  "type": ["string", "null"],
+                  "description": "The parent container name or null for the root container."
+                },
+                "owner": {
+                  "type": ["integer", "null"],
+                  "description": "The thread identifier of the owner thread or null if not owned."
+                },
+                "threads": {
+                  "type": "array",
+                  "description": "The array of threads in the thread container.",
+                  "items": [
+                    {
+                      "type": "object",
+                      "properties": {
+                        "tid": {
+                          "type": "integer",
+                          "description": "The thread identifier, see java.lang.Thread.threadId()."
+                        },
+                        "name": {
+                          "type": "string",
+                          "description": "The thread name."
+                        },
+                        "stack": {
+                          "type": "array",
+                          "description": "The thread stack. The first element is the top of the stack.",
+                          "items": [
+                            {
+                              "type": "string",
+                              "description": "A stack trace element, see java.lang.StackTraceElement."
+                            }
+                          ]
+                        }
+                      },
+                      "required": [
+                        "tid",
+                        "name",
+                        "stack"
+                      ]
+                    }
+                  ]
+                },
+                "threadCount": {
+                  "type": "integer",
+                  "description": "The number of threads in the thread container."
+                }
+              },
+              "required": [
+                "container",
+                "parent",
+                "owner",
+                "threads",
+                "threadCount"
+              ]
+            }
+          ]
+        }
+      },
+      "required": [
+        "processId",
+        "time",
+        "runtimeVersion",
+        "threadContainers"
+      ]
+    }
+  },
+  "required": [
+    "threadDump"
+  ]
+}
```

Comment:
This should probably include a trailing new‑line, and the file should use four spaces for indentation.